l10n_it_ipa/model/l10n_it_ipa_mixin.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import models, fields
+
+
+class L10nItIpaMixin(models.AbstractModel):
+    _name = 'l10n_it_ipa.mixin'
```

Comment:
Why another mixin in addition to `l10n_it_account.mixin`?

Reply:
Because this addon haven't `l10n_it_account` dependency.

Reply:
Right. Then maybe we should have a basic module (`l10n_it_multi_country`?) that introduces the mixin and make all these modules depend on it.

@pedrobaeza what do you think? Do you have anything similar in Spain?

@OCA/local-italy-maintainers cheddite?

Reply:
Probably this should designed at OCA level, not only for `l10n-italy`

Reply:
Yeah, `l10n_it_base` containing the mixin is the best option, but not everyone will like to add on this version a new module dependency in the middle of the version (I would be one of them being on the opposite side).

Reply:
Not bad idea, although in Spain we work at CoA level, not country code level. By the way, do you have a link to a complete Italian CoA that you use instead of the official one?

Reply:
I don't use this CoA for many customers, perhaps only some migrated, but for e.g.: https://github.com/sergiocorato/efatto/tree/12.0/l10n_it_generic
I agree with a module `l10n_multi_country` at OCA level, to add as dependency in modules of whatever localization, or to specific newly created modules inheriting the standard modules.